etl-data-pipeline/src/main/resources/db/sp-organization-001.sql

```diff
@@ -1,4 +1,4 @@
-CREATE PROCEDURE [dbo].[sp_Organization]
+CREATE or Alter PROCEDURE [dbo].[sp_Organization]
```

Comment:
This will be maintained by liquibase, correct?

Reply:
Yeah if you have to make any changes going forward, you will have to create a new revision file such as:
sp-organization-002.sql